.github/workflows/aks_deploy.yml

```diff
@@ -26,7 +26,7 @@ jobs:
   docker:
     name: Build and push Docker image
     runs-on: ubuntu-latest
-    if: contains(github.event.pull_request.labels.*.name, 'review-app')
+    if: contains(github.event.pull_request.labels.*.name, 'review-app') || github.event_name != 'pull_request'
```

Comment:
```suggestion
    if: contains(github.event.pull_request.labels.*.name, 'review-app') || github.ref == 'refs/heads/main'
```

Would this work? 🤔

Reply:
I think you will always want the docker step to run i.e. on pr update (with or without review-app), and merge to main. So the if statement would be dropped.

Reply:
Also deploy-review if should be on pr only, so line 46 `if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'review-app') }}`